### PR TITLE
220112 add dns debug

### PR DESCRIPTION
**Pull request by pillip8282**

### CODE REVIEW

File: apps/examples/wifi_manager/wm_test/wm_test_dns.c [hunk 1]
```diff
@@ -0,0 +1,119 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
```
Comment (jeongchanKim) [inline apps/examples/wifi_manager/wm_test/wm_test_dns.c:3]: Please change to 2022.

File: apps/examples/wifi_manager/wm_test/wm_test_dns.c [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+		WT_LOGP(TAG, "%d.%d.%d.%d\n", addr[0], addr[1], addr[2], addr[3]);
+	}
+	WT_LOGP(TAG, "\n");
+}
+
+START_TEST_F(dns_queue)
+{
+	struct addrinfo hints, *local_res;
+	int arr_size = sizeof(g_host_list) / sizeof(char *);
+	memset(&hints, 0, sizeof(hints));
```
Comment (jeongchanKim) [inline apps/examples/wifi_manager/wm_test/wm_test_dns.c:83]: I think `sizeof(struct addrinfo)` is better.

Comment (pillip8282) [reply apps/examples/wifi_manager/wm_test/wm_test_dns.c:83]: Thanks I'll modify it

File: apps/examples/wifi_manager/wm_test/wm_test_dns.c (hunk 1 above)
Comment (pillip8282) [reply apps/examples/wifi_manager/wm_test/wm_test_dns.c:3]: Thanks I'll modify it

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**